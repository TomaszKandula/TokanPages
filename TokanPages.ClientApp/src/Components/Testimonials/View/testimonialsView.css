.testimonials-card-image {
    height: 140px !important;
    width: 140px !important;
    border-style: solid;
    border-width: 4px;
    border-color: #6367ef;
    border-radius: 50%;
    margin: 0;
    left: 50%;
    transform: translate(-50%, -50%);
    overflow: hidden;
    position: absolute;
}

.testimonials-card-columns {
    margin: 3rem !important;
}

@media (max-width: 768px) {
    .testimonials-card-columns {
        margin: 0.5rem !important;
    }
}

.testimonials-card-column {
    margin-top: 0 !important;
}

@media (max-width: 768px) {
    .testimonials-card-column {
        margin-top: 3rem !important;
    }
}

.testimonials-card-content {
    padding: 1rem;
}

@media (min-width: 768px) and (max-width: 1024px) {
    .testimonials-card-content {
        padding-top: 1rem;
        padding-bottom: 1rem;
        padding-right: 0;
        padding-left: 0;
    }
}
