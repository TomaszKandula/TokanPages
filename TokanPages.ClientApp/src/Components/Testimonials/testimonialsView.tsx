import * as React from "react";
import Container from "@material-ui/core/Container";
import Box from "@material-ui/core/Box";
import Grid from "@material-ui/core/Grid";
import Card from "@material-ui/core/Card";
import CardMedia from "@material-ui/core/CardMedia";
import CardContent from "@material-ui/core/CardContent";
import Typography from "@material-ui/core/Typography";
import Skeleton from "@material-ui/lab/Skeleton";
import { IGetTestimonialsContent } from "../../Redux/States/Content/getTestimonialsContentState";
import { TESTIMONIALS_PATH } from "../../Shared/constants";
import { GetShortText } from "../../Shared/helpers";
import testimonialsStyle from "./testimonialsStyle";
import "react-responsive-carousel/lib/styles/carousel.min.css";
import "../../Theme/CustomCss/overrideCarousel.css";

const TestimonialsView = (props: IGetTestimonialsContent): JSX.Element =>
{
    const classes = testimonialsStyle();
    const limit = 29;
    const imageUrl = (name: string) => 
    {
        if (name === "") return " ";
        return `${TESTIMONIALS_PATH}${name}`;
    }

    return(
        <section className={classes.section}>
            <Container maxWidth="lg"> 
                <Box pt={8} pb={10} textAlign="center" mb={5}>
                    <Typography gutterBottom={true} className={classes.caption_text} data-aos="fade-down">
                        {props.isLoading ? <Skeleton variant="text" /> : props.content?.caption?.toUpperCase()}
                    </Typography>
                </Box>
                <Box pb={15} textAlign="center">
                    <Grid container spacing={6}>
                        <Grid item xs={12} md={4} data-aos="fade-left">
                            <Card elevation={0} className={classes.card}>
                                <CardMedia image={imageUrl(props.content?.photo1)} component="img" className={classes.card_image} />
                                <CardContent className={classes.card_content}>
                                    <Typography className={classes.card_title}>
                                        {props.content?.name1}
                                    </Typography>
                                    <Typography className={classes.card_subheader}>
                                        {props.content?.occupation1}
                                    </Typography>
                                    <Typography className={classes.card_text}>
                                        {GetShortText(props.content?.text1, limit)}
                                    </Typography>
                                </CardContent>
                            </Card>
                        </Grid>
                        <Grid item xs={12} md={4} data-aos="fade-up">
                            <Card elevation={3} className={classes.card}>
                                <CardMedia image={imageUrl(props.content?.photo2)} component="img" className={classes.card_image} />
                                <CardContent className={classes.card_content}>
                                <Typography className={classes.card_title}>
                                        {props.content?.name2}
                                    </Typography>
                                    <Typography className={classes.card_subheader}>
                                        {props.content?.occupation2}
                                    </Typography>
                                    <Typography className={classes.card_text}>
                                        {GetShortText(props.content?.text2, limit)}
                                    </Typography>
                                </CardContent>
                            </Card>
                        </Grid>
                        <Grid item xs={12} md={4} data-aos="fade-right">
                            <Card elevation={3} className={classes.card}>
                                <CardMedia image={imageUrl(props.content?.photo3)} component="img" className={classes.card_image} />
                                <CardContent className={classes.card_content}>
                                <Typography className={classes.card_title}>
                                        {props.content?.name3}
                                    </Typography>
                                    <Typography className={classes.card_subheader}>
                                        {props.content?.occupation3}
                                    </Typography>
                                    <Typography className={classes.card_text}>
                                        {GetShortText(props.content?.text3, limit)}
                                    </Typography>
                                </CardContent>
                            </Card>
                        </Grid>
                    </Grid>
                </Box>
            </Container>
        </section>
    );
}

export default TestimonialsView;
