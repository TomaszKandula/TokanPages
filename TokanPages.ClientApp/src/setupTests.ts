// jest-dom adds custom jest matchers for asserting on DOM nodes.
// allows you to do things like:
// expect(element).toHaveTextContent(/react/i)
// learn more: https://github.com/testing-library/jest-dom
import "@testing-library/jest-dom";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import enableHooks from "jest-react-hooks-shallow";

enableHooks(jest);

jest.mock("react-redux", () => (
{
    useSelector: jest.fn(),
    useDispatch: jest.fn()
}));

jest.mock("connected-react-router", () => (
{
    connectRouter: jest.fn(),
    routerMiddleware: jest.fn()
}));

Enzyme.configure({ adapter: new Adapter(), disableLifecycleMethods: false });

// environmental variables used during all tests
process.env.REACT_APP_API_VER="1";
process.env.REACT_APP_BACKEND="http://localhost:5000";
