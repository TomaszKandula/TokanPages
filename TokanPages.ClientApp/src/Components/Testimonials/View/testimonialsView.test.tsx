import "../../../setupTests";
import React from "react";
import { render } from "enzyme";
import { TestimonialsView } from "./testimonialsView";

describe("test component: testimonialsView", () => {
    const testContent = {
        language: "eng",
        caption: "Testimonials",
        subtitle: "You can read few commendations...",
        photo1: "",
        name1: "Joanna",
        occupation1: "Senior Digital Tax Specialist",
        text1: "Joe is very motivated...",
        photo2: "",
        name2: "Adama",
        occupation2: "Full-stack Developer chez DFDS POLSKA",
        text2: "Joe have done very good work...",
        photo3: "",
        name3: "Scott",
        occupation3: "BPO",
        text3: "Joe is one of those rare talents...",
    };

    it("should render correctly '<TestimonialsView />' when content is loaded.", () => {
        const html = render(<TestimonialsView 
            isLoading={false}
            hasTestimonialOne={true}
            hasTestimonialTwo={true}
            hasTestimonialThree={true}
            buttonTestimonialOne={jest.fn()}
            buttonTestimonialTwo={jest.fn()}
            buttonTestimonialThree={jest.fn()}
            caption={testContent.caption}
            subtitle={testContent.subtitle}
            photo1={testContent.photo1}
            name1={testContent.name1}
            occupation1={testContent.occupation1}
            text1={testContent.text1}
            photo2={testContent.photo2}
            name2={testContent.name2}
            occupation2={testContent.occupation2}
            text2={testContent.text2}
            photo3={testContent.name3}
            name3={testContent.name3}
            occupation3={testContent.occupation3}
            text3={testContent.text3}
        />);

        expect(html).toMatchSnapshot();
    });
});
