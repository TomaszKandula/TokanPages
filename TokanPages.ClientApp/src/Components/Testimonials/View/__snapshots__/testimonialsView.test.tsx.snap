// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component: testimonialsView should render correctly '<TestimonialsView />' when content is loaded. 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section>
        <div
          class="bulma-container"
        >
          <div
            class="py-6"
          >
            <div
              data-aos="fade-down"
              data-aos-delay="0"
            >
              <p
                class="is-size-3	has-text-centered has-text-link"
              >
                TESTIMONIALS
              </p>
            </div>
            <div
              class="bulma-columns testimonials-card-columns"
            >
              <div
                class="bulma-column testimonials-card-column"
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="350"
                >
                  <div
                    class="bulma-card"
                  >
                    <img
                      alt="Picture of Joanna"
                      class="testimonials-card-image"
                      loading="lazy"
                      src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/joanna.webp"
                      title="Testimonials"
                    />
                    <div
                      class="bulma-card-content mt-6 testimonials-card-content"
                    >
                      <h3
                        class="is-size-5 has-text-centered has-text-weight-semibold mt-6 px-4 pt-4 pb-1"
                      >
                        Joanna
                      </h3>
                      <div
                        class="is-flex is-justify-content-center"
                      >
                        <a
                          class="href "
                          href="https://www.linkedin.com/in/joanna"
                          rel="noopener nofollow"
                          target="_blank"
                        >
                          <figure
                            class="bulma-image bulma-is-24x24"
                          >
                            <svg
                              class="is-flex is-align-self-center "
                              fill="#000000"
                              style="width: 1.5rem; height: 1.5rem;"
                              version="1.1"
                              viewBox="0 0 32 32"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                              />
                            </svg>
                          </figure>
                        </a>
                      </div>
                      <p
                        class="is-size-6 has-text-centered has-text-link p-2"
                      >
                        Senior Digital Tax Specialist
                      </p>
                      <div
                        class="is-flex is-flex-direction-column is-align-items-center"
                      >
                        <div
                          class="collapsible-transition"
                          style="height: 120px;"
                        >
                          <div
                            class="collapsible-content"
                          >
                            <div
                              class="collapsible-gradient"
                              style="height: 120px;"
                            />
                            <p
                              class="is-size-6 has-text-centered has-text-grey p-2 line-height-18"
                            >
                              Joe is very motivated...
                            </p>
                          </div>
                        </div>
                        <button
                          class="icon-button collapsible-expand is-flex is-align-self-center"
                          style="height: 48px; width: 48px;"
                        >
                          <div>
                            <svg
                              class="is-flex is-align-self-center has-text-grey"
                              role="presentation"
                              style="width: 1.5rem; height: 1.5rem;"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                                style="fill: currentColor;"
                              />
                            </svg>
                          </div>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="bulma-column testimonials-card-column"
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="150"
                >
                  <div
                    class="bulma-card"
                  >
                    <img
                      alt="Picture of Adama"
                      class="testimonials-card-image"
                      loading="lazy"
                      src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/adama.webp"
                      title="Testimonials"
                    />
                    <div
                      class="bulma-card-content mt-6 testimonials-card-content"
                    >
                      <h3
                        class="is-size-5 has-text-centered has-text-weight-semibold mt-6 px-4 pt-4 pb-1"
                      >
                        Adama
                      </h3>
                      <div
                        class="is-flex is-justify-content-center"
                      >
                        <a
                          class="href "
                          href="https://www.linkedin.com/in/adama"
                          rel="noopener nofollow"
                          target="_blank"
                        >
                          <figure
                            class="bulma-image bulma-is-24x24"
                          >
                            <svg
                              class="is-flex is-align-self-center "
                              fill="#000000"
                              style="width: 1.5rem; height: 1.5rem;"
                              version="1.1"
                              viewBox="0 0 32 32"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                              />
                            </svg>
                          </figure>
                        </a>
                      </div>
                      <p
                        class="is-size-6 has-text-centered has-text-link p-2"
                      >
                        Full-stack Developer chez DFDS POLSKA
                      </p>
                      <div
                        class="is-flex is-flex-direction-column is-align-items-center"
                      >
                        <div
                          class="collapsible-transition"
                        >
                          <div
                            class="collapsible-content"
                          >
                            <div
                              class="collapsible-gradient"
                            />
                            <p
                              class="is-size-6 has-text-centered has-text-grey p-2 line-height-18"
                            >
                              Joe have done very good work...
                            </p>
                          </div>
                        </div>
                        <button
                          class="icon-button collapsible-expand is-flex is-align-self-center"
                          style="height: 48px; width: 48px;"
                        >
                          <div>
                            <svg
                              class="is-flex is-align-self-center has-text-grey"
                              role="presentation"
                              style="width: 1.5rem; height: 1.5rem;"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                                style="fill: currentColor;"
                              />
                            </svg>
                          </div>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="bulma-column testimonials-card-column"
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="250"
                >
                  <div
                    class="bulma-card"
                  >
                    <img
                      alt="Picture of Scott"
                      class="testimonials-card-image"
                      loading="lazy"
                      src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/scott.webp"
                      title="Testimonials"
                    />
                    <div
                      class="bulma-card-content mt-6 testimonials-card-content"
                    >
                      <h3
                        class="is-size-5 has-text-centered has-text-weight-semibold mt-6 px-4 pt-4 pb-1"
                      >
                        Scott
                      </h3>
                      <div
                        class="is-flex is-justify-content-center"
                      >
                        <a
                          class="href "
                          href="https://www.linkedin.com/in/scott"
                          rel="noopener nofollow"
                          target="_blank"
                        >
                          <figure
                            class="bulma-image bulma-is-24x24"
                          >
                            <svg
                              class="is-flex is-align-self-center "
                              fill="#000000"
                              style="width: 1.5rem; height: 1.5rem;"
                              version="1.1"
                              viewBox="0 0 32 32"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                              />
                            </svg>
                          </figure>
                        </a>
                      </div>
                      <p
                        class="is-size-6 has-text-centered has-text-link p-2"
                      >
                        BPO
                      </p>
                      <div
                        class="is-flex is-flex-direction-column is-align-items-center"
                      >
                        <div
                          class="collapsible-transition"
                        >
                          <div
                            class="collapsible-content"
                          >
                            <div
                              class="collapsible-gradient"
                            />
                            <p
                              class="is-size-6 has-text-centered has-text-grey p-2 line-height-18"
                            >
                              Joe is one of those rare talents...
                            </p>
                          </div>
                        </div>
                        <button
                          class="icon-button collapsible-expand is-flex is-align-self-center"
                          style="height: 48px; width: 48px;"
                        >
                          <div>
                            <svg
                              class="is-flex is-align-self-center has-text-grey"
                              role="presentation"
                              style="width: 1.5rem; height: 1.5rem;"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                                style="fill: currentColor;"
                              />
                            </svg>
                          </div>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section>
      <div
        class="bulma-container"
      >
        <div
          class="py-6"
        >
          <div
            data-aos="fade-down"
            data-aos-delay="0"
          >
            <p
              class="is-size-3	has-text-centered has-text-link"
            >
              TESTIMONIALS
            </p>
          </div>
          <div
            class="bulma-columns testimonials-card-columns"
          >
            <div
              class="bulma-column testimonials-card-column"
            >
              <div
                data-aos="fade-up"
                data-aos-delay="350"
              >
                <div
                  class="bulma-card"
                >
                  <img
                    alt="Picture of Joanna"
                    class="testimonials-card-image"
                    loading="lazy"
                    src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/joanna.webp"
                    title="Testimonials"
                  />
                  <div
                    class="bulma-card-content mt-6 testimonials-card-content"
                  >
                    <h3
                      class="is-size-5 has-text-centered has-text-weight-semibold mt-6 px-4 pt-4 pb-1"
                    >
                      Joanna
                    </h3>
                    <div
                      class="is-flex is-justify-content-center"
                    >
                      <a
                        class="href "
                        href="https://www.linkedin.com/in/joanna"
                        rel="noopener nofollow"
                        target="_blank"
                      >
                        <figure
                          class="bulma-image bulma-is-24x24"
                        >
                          <svg
                            class="is-flex is-align-self-center "
                            fill="#000000"
                            style="width: 1.5rem; height: 1.5rem;"
                            version="1.1"
                            viewBox="0 0 32 32"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                            />
                          </svg>
                        </figure>
                      </a>
                    </div>
                    <p
                      class="is-size-6 has-text-centered has-text-link p-2"
                    >
                      Senior Digital Tax Specialist
                    </p>
                    <div
                      class="is-flex is-flex-direction-column is-align-items-center"
                    >
                      <div
                        class="collapsible-transition"
                        style="height: 120px;"
                      >
                        <div
                          class="collapsible-content"
                        >
                          <div
                            class="collapsible-gradient"
                            style="height: 120px;"
                          />
                          <p
                            class="is-size-6 has-text-centered has-text-grey p-2 line-height-18"
                          >
                            Joe is very motivated...
                          </p>
                        </div>
                      </div>
                      <button
                        class="icon-button collapsible-expand is-flex is-align-self-center"
                        style="height: 48px; width: 48px;"
                      >
                        <div>
                          <svg
                            class="is-flex is-align-self-center has-text-grey"
                            role="presentation"
                            style="width: 1.5rem; height: 1.5rem;"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                              style="fill: currentColor;"
                            />
                          </svg>
                        </div>
                      </button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="bulma-column testimonials-card-column"
            >
              <div
                data-aos="fade-up"
                data-aos-delay="150"
              >
                <div
                  class="bulma-card"
                >
                  <img
                    alt="Picture of Adama"
                    class="testimonials-card-image"
                    loading="lazy"
                    src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/adama.webp"
                    title="Testimonials"
                  />
                  <div
                    class="bulma-card-content mt-6 testimonials-card-content"
                  >
                    <h3
                      class="is-size-5 has-text-centered has-text-weight-semibold mt-6 px-4 pt-4 pb-1"
                    >
                      Adama
                    </h3>
                    <div
                      class="is-flex is-justify-content-center"
                    >
                      <a
                        class="href "
                        href="https://www.linkedin.com/in/adama"
                        rel="noopener nofollow"
                        target="_blank"
                      >
                        <figure
                          class="bulma-image bulma-is-24x24"
                        >
                          <svg
                            class="is-flex is-align-self-center "
                            fill="#000000"
                            style="width: 1.5rem; height: 1.5rem;"
                            version="1.1"
                            viewBox="0 0 32 32"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                            />
                          </svg>
                        </figure>
                      </a>
                    </div>
                    <p
                      class="is-size-6 has-text-centered has-text-link p-2"
                    >
                      Full-stack Developer chez DFDS POLSKA
                    </p>
                    <div
                      class="is-flex is-flex-direction-column is-align-items-center"
                    >
                      <div
                        class="collapsible-transition"
                      >
                        <div
                          class="collapsible-content"
                        >
                          <div
                            class="collapsible-gradient"
                          />
                          <p
                            class="is-size-6 has-text-centered has-text-grey p-2 line-height-18"
                          >
                            Joe have done very good work...
                          </p>
                        </div>
                      </div>
                      <button
                        class="icon-button collapsible-expand is-flex is-align-self-center"
                        style="height: 48px; width: 48px;"
                      >
                        <div>
                          <svg
                            class="is-flex is-align-self-center has-text-grey"
                            role="presentation"
                            style="width: 1.5rem; height: 1.5rem;"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                              style="fill: currentColor;"
                            />
                          </svg>
                        </div>
                      </button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="bulma-column testimonials-card-column"
            >
              <div
                data-aos="fade-up"
                data-aos-delay="250"
              >
                <div
                  class="bulma-card"
                >
                  <img
                    alt="Picture of Scott"
                    class="testimonials-card-image"
                    loading="lazy"
                    src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/scott.webp"
                    title="Testimonials"
                  />
                  <div
                    class="bulma-card-content mt-6 testimonials-card-content"
                  >
                    <h3
                      class="is-size-5 has-text-centered has-text-weight-semibold mt-6 px-4 pt-4 pb-1"
                    >
                      Scott
                    </h3>
                    <div
                      class="is-flex is-justify-content-center"
                    >
                      <a
                        class="href "
                        href="https://www.linkedin.com/in/scott"
                        rel="noopener nofollow"
                        target="_blank"
                      >
                        <figure
                          class="bulma-image bulma-is-24x24"
                        >
                          <svg
                            class="is-flex is-align-self-center "
                            fill="#000000"
                            style="width: 1.5rem; height: 1.5rem;"
                            version="1.1"
                            viewBox="0 0 32 32"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                            />
                          </svg>
                        </figure>
                      </a>
                    </div>
                    <p
                      class="is-size-6 has-text-centered has-text-link p-2"
                    >
                      BPO
                    </p>
                    <div
                      class="is-flex is-flex-direction-column is-align-items-center"
                    >
                      <div
                        class="collapsible-transition"
                      >
                        <div
                          class="collapsible-content"
                        >
                          <div
                            class="collapsible-gradient"
                          />
                          <p
                            class="is-size-6 has-text-centered has-text-grey p-2 line-height-18"
                          >
                            Joe is one of those rare talents...
                          </p>
                        </div>
                      </div>
                      <button
                        class="icon-button collapsible-expand is-flex is-align-self-center"
                        style="height: 48px; width: 48px;"
                      >
                        <div>
                          <svg
                            class="is-flex is-align-self-center has-text-grey"
                            role="presentation"
                            style="width: 1.5rem; height: 1.5rem;"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                              style="fill: currentColor;"
                            />
                          </svg>
                        </div>
                      </button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
